### DISCUSSION TITLE
override thermostat value steps possible?

### DISCUSSION THREAD

**Discussion by derSchreifritz** [emsesp/EMS-ESP32#900] [Q&A]
when I type 19.4 in hp manual temperature, it changes to 19.0. Is it possible to have steps of 0.1 accepted? Or is it a thermostat thing that we cannot change?

**Comment by proddy** [comment]
probably a thermostat thing. Can you dial in 19.4 on the thermostat?

**Reply by derSchreifritz** [reply]
no. Only in steps of 0.5
It displays the actual room temperature also in steps of 0.5, but through EMS I can read it in steps of 0.1. So I was hoping the limitation is just for the display.
